### ISSUE TITLE
Published npm package does not appear in npm info and can't be installed

### ISSUE THREAD

**Issue by voltrevo** [npm/npm#12738]
My build server apparently published this package several hours ago:

```
+ npm publish
+ @opentok/opentok-plugin@1.0.1-alpha.269
```

but it doesn't appear to actually have been published:

```
$ npm info \@opentok/opentok-plugin | grep 269
$
```

and I cannot install it:

```
$ cd /tmp
$ mkdir -p node_modules
$ npm install \@opentok/opentok-plugin@1.0.1-alpha.269
npm ERR! Darwin 15.4.0
npm ERR! argv "/Users/andrew/.node_stuff/node-v5.1.0-darwin-x64/bin/node" "/Users/andrew/.node_stuff/curr/bin/npm" "install" "@opentok/opentok-plugin@1.0.1-alpha.269"
npm ERR! node v5.1.0
npm ERR! npm  v2.14.12

npm ERR! version not found: @opentok/opentok-plugin@1.0.1-alpha.269
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /private/tmp/npm-debug.log
$ cat /private/tmp/npm-debug.log 
0 info it worked if it ends with ok
1 verbose cli [ '/Users/andrew/.node_stuff/node-v5.1.0-darwin-x64/bin/node',
1 verbose cli   '/Users/andrew/.node_stuff/curr/bin/npm',
1 verbose cli   'install',
1 verbose cli   '@opentok/opentok-plugin@1.0.1-alpha.269' ]
2 info using npm@2.14.12
3 info using node@v5.1.0
4 verbose install initial load of /private/tmp/package.json
5 verbose installManyTop reading scoped package data from /private/tmp/node_modules/@opentok/otplugin.js/package.json
6 verbose readDependencies loading dependencies from /private/tmp/package.json
7 silly cache add args [ '@opentok/opentok-plugin@1.0.1-alpha.269', null ]
8 verbose cache add spec @opentok/opentok-plugin@1.0.1-alpha.269
9 silly cache add parsed spec Result {
9 silly cache add   raw: '@opentok/opentok-plugin@1.0.1-alpha.269',
9 silly cache add   scope: '@opentok',
9 silly cache add   name: '@opentok/opentok-plugin',
9 silly cache add   rawSpec: '1.0.1-alpha.269',
9 silly cache add   spec: '1.0.1-alpha.269',
9 silly cache add   type: 'version' }
10 silly addNamed @opentok/opentok-plugin@1.0.1-alpha.269
11 verbose addNamed "1.0.1-alpha.269" is a plain semver version for @opentok/opentok-plugin
12 silly mapToRegistry name @opentok/opentok-plugin
13 silly mapToRegistry scope (from package name) @opentok
14 verbose mapToRegistry no registry URL found in name for scope @opentok
15 silly mapToRegistry using default registry
16 silly mapToRegistry registry https://registry.npmjs.org/
17 silly mapToRegistry uri https://registry.npmjs.org/@opentok%2fopentok-plugin
18 verbose addNameVersion registry:https://registry.npmjs.org/@opentok%2fopentok-plugin not in flight; fetching
19 verbose request uri https://registry.npmjs.org/@opentok%2fopentok-plugin
20 verbose request no auth needed
21 info attempt registry request try #1 at 12:17:53 PM
22 verbose request using bearer token for auth
23 verbose request id 4781e465540b7318
24 verbose etag "DSV0KMFWK1AMPHN3LZ4N35RMB"
25 http request GET https://registry.npmjs.org/@opentok%2fopentok-plugin
26 http 404 https://registry.npmjs.org/@opentok%2fopentok-plugin
27 verbose headers { 'content-type': 'application/json',
27 verbose headers   'cache-control': 'max-age=0',
27 verbose headers   'content-length': '21',
27 verbose headers   'accept-ranges': 'bytes',
27 verbose headers   date: 'Wed, 18 May 2016 02:17:53 GMT',
27 verbose headers   via: '1.1 varnish',
27 verbose headers   connection: 'keep-alive',
27 verbose headers   'x-served-by': 'cache-syd1625-SYD',
27 verbose headers   'x-cache': 'MISS',
27 verbose headers   'x-cache-hits': '0',
27 verbose headers   'x-timer': 'S1463537873.462111,VS0,VE449' }
28 silly get cb [ 304, undefined ]
29 verbose etag https://registry.npmjs.org/@opentok%2fopentok-plugin from cache
30 verbose get saving @opentok/opentok-plugin to /Users/andrew/.npm/registry.npmjs.org/_40opentok_252fopentok-plugin/.cache.json
31 verbose stack Error: version not found: @opentok/opentok-plugin@1.0.1-alpha.269
31 verbose stack     at setData (/Users/andrew/.node_stuff/node-v5.1.0-darwin-x64/lib/node_modules/npm/lib/cache/add-named.js:137:12)
31 verbose stack     at RES (/Users/andrew/.node_stuff/node-v5.1.0-darwin-x64/lib/node_modules/npm/node_modules/inflight/inflight.js:23:14)
31 verbose stack     at f (/Users/andrew/.node_stuff/node-v5.1.0-darwin-x64/lib/node_modules/npm/node_modules/once/once.js:17:25)
31 verbose stack     at fixName (/Users/andrew/.node_stuff/node-v5.1.0-darwin-x64/lib/node_modules/npm/lib/cache/add-named.js:29:5)
31 verbose stack     at saved (/Users/andrew/.node_stuff/node-v5.1.0-darwin-x64/lib/node_modules/npm/lib/cache/caching-client.js:173:7)
31 verbose stack     at /Users/andrew/.node_stuff/node-v5.1.0-darwin-x64/lib/node_modules/npm/node_modules/graceful-fs/polyfills.js:210:7
31 verbose stack     at FSReqWrap.oncomplete (fs.js:82:15)
32 verbose statusCode 404
33 verbose cwd /private/tmp
34 error Darwin 15.4.0
35 error argv "/Users/andrew/.node_stuff/node-v5.1.0-darwin-x64/bin/node" "/Users/andrew/.node_stuff/curr/bin/npm" "install" "@opentok/opentok-plugin@1.0.1-alpha.269"
36 error node v5.1.0
37 error npm  v2.14.12
38 error version not found: @opentok/opentok-plugin@1.0.1-alpha.269
39 error If you need help, you may report this error at:
39 error     <https://github.com/npm/npm/issues>
40 verbose exit [ 1, true ]
```

**Comment by germanattanasio** [comment]
I'm having the same issue, I tried to deploy a new version

```none
[germana:~/workspace/sunburst-chart]$ npm publish                                                                                         (master)
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts will run only for `npm publish`.
npm WARN prepublish-on-install (In npm@4 and previous versions, it also runs for `npm install`.)
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.

> personality-sunburst-chart@2.1.1 prepublish /Users/germana/workspace/sunburst-chart
> npm run compile && npm run build


> personality-sunburst-chart@2.1.1 compile /Users/germana/workspace/sunburst-chart
> browserify --full-paths -t [ babelify --presets [ es2015 ] ] --standalone PersonalitySunburstChart src/index.js -o dist/index.js


> personality-sunburst-chart@2.1.1 build /Users/germana/workspace/sunburst-chart
> babel src --presets es2015 --out-dir lib

src/d3-profile-wrappers/v2/index.js -> lib/d3-profile-wrappers/v2/index.js
src/d3-profile-wrappers/v3/index.js -> lib/d3-profile-wrappers/v3/index.js
src/index.js -> lib/index.js
src/personality-chart-renderer.js -> lib/personality-chart-renderer.js
src/utilities/colors.js -> lib/utilities/colors.js
+ personality-sunburst-chart@2.1.1
```

Everything looks good but then when I run `npm show`

```none
[germana:~/workspace/sunburst-chart]$ npm info personality-sunburst-chart                                                                 (master)

{ name: 'personality-sunburst-chart',
  description: 'Generate Sunburst Chart for Personality Insights Profile',
  'dist-tags': { latest: '2.0.5' },
  versions: [ '1.0.0', '1.0.1', '1.0.2', '2.0.2', '2.0.4', '2.0.5' ],
  maintainers:
   [ 'agustincasamayor <agustincasamayor@gmail.com>',
     'aprilwebster <awebster@us.ibm.com>',
     'arypbatista <arypbatista@gmail.com>',
     'neil-boyette-ibm <neil.boyette@us.ibm.com>',
     'vibhasinha <vibha.singhal@gmail.com>' ],
  time:
   { modified: '2017-03-02T15:55:47.057Z',
     created: '2016-02-12T12:37:26.611Z',
     '1.0.0': '2016-02-12T12:37:26.611Z',
     '1.0.1': '2016-11-08T02:47:24.893Z',
     '1.0.2': '2016-11-12T22:07:37.224Z',
     '2.0.2': '2017-01-25T23:25:39.822Z',
     '2.0.4': '2017-01-27T00:59:47.247Z',
     '2.0.5': '2017-03-02T15:55:47.057Z' },
  homepage: 'https://github.com/personality-insights/sunburst-chart#readme',
  keywords:
   [ 'personality',
     'insights',
     'sunburst',
     'chart',
     'visualization',
     'ibm',
     'watson' ],
  repository:
   { type: 'git',
     url: 'git+ssh://git@github.com/personality-insights/sunburst-chart.git' },
  author: 'IBM Corp.',
  bugs: { url: 'https://github.com/personality-insights/sunburst-chart/issues' },
  license: 'Apache-2.0',
  readmeFilename: 'README.md',
  version: '2.0.5',
  main: 'lib/index.js',
  exportName: 'PersonalitySunburstChart',
  scripts:
   { prepublish: 'npm run compile && npm run build',
     compile: 'browserify --full-paths -t [ babelify --presets [ es2015 ] ] --standalone PersonalitySunburstChart src/index.js -o dist/index.js',
     build: 'babel src --presets es2015 --out-dir lib',
     test: 'npm run lint && npm run codecov',
     lint: 'eslint .',
     autofix: 'eslint . --fix',
     codecov: 'istanbul cover mocha test && codecov' },
  engines: { node: '>4.0' },
  dependencies: { extend: '^3.0.0', 'object.pick': '^1.2.0' },
  devDependencies:
   { browserify: '^14.0.0',
     'babel-preset-es2015': '^6.18.0',
     'babel-cli': '^6.18.0',
     babelify: '^7.3.0',
     chai: '^3.5.0',
     codecov: '^1.0.1',
     eslint: '^3.11.1',
     istanbul: '^0.4.3',
     mocha: '^3.2.0' },
  gitHead: 'bc0305e3633edf6f5bd0d037fc0141424796eb32',
  dist:
   { shasum: '4d407a622d4e3104f0a6259b279d438480277841',
     tarball: 'https://npm-registry.whitewater.ibm.com/p/personality-sunburst-chart/_attachments/personality-sunburst-chart-2.0.5.tgz' },
  directories: {} }
```

`2.1.1` is not there. 📦

**Comment by legodude17** [comment]
Can you publish over it?

**Comment by voltrevo** [comment]
@legodude17 for me this issue is no longer relevant (didn't find out what the problem was though.

I think part of the issue may have been we were using npm to publish several multi-megabyte binary files. @germanattanasio perhaps this applies for you too?

**Comment by germanattanasio** [comment]
@legodude17 this is the result of publishing the same version again
```
[germana:~/workspace/sunburst-chart]$ npm publish                                                             (master)
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts will run only for `npm publish`.
npm WARN prepublish-on-install (In npm@4 and previous versions, it also runs for `npm install`.)
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.

> personality-sunburst-chart@2.1.2 prepublish /Users/germana/workspace/sunburst-chart
> npm run compile && npm run build


> personality-sunburst-chart@2.1.2 compile /Users/germana/workspace/sunburst-chart
> browserify --full-paths -t [ babelify --presets [ es2015 ] ] --standalone PersonalitySunburstChart src/index.js -o dist/index.js


> personality-sunburst-chart@2.1.2 build /Users/germana/workspace/sunburst-chart
> babel src --presets es2015 --out-dir lib

src/d3-profile-wrappers/v2/index.js -> lib/d3-profile-wrappers/v2/index.js
src/d3-profile-wrappers/v3/index.js -> lib/d3-profile-wrappers/v3/index.js
src/index.js -> lib/index.js
src/personality-chart-renderer.js -> lib/personality-chart-renderer.js
src/utilities/colors.js -> lib/utilities/colors.js
npm ERR! publish Failed PUT 403
npm ERR! code E403
npm ERR! "You cannot publish over the previously published version 2.1.2." : personality-sunburst-chart

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/germana/.npm/_logs/2017-05-12T02_04_59_103Z-debug.log
```

@voltrevo my package is composed of just javascript files.
